Remarkable-Donkeys/bangazon
# Add CORS

# Steps To Test #
1. `git fetch --all`
2. `git checkout addCORS`
3. add "bangazon.com" and "www.bangazon.com" as local host aliases
Do this (on Mac) by:
   1.`sudo nano /etc/hosts`
   2. under `127.0.0.1  localhost` add `127.0.0.1 bangazon.com` and then `127.0.0.1 www.bangazon.com`
*make sure everything lines up so they look like the aliases that are already there
   3. exit the file and confirm changes
4. if not already doing so run the server `dotnet run`
5. open browser and run bangazon.com:5000/api/customer

[Bangazonian Access Only](https://github.com/Remarkable-Donkeys/bangazon/issues/12)

### Description of what should happen when steps are followed ###
After all the steps are followed, it should return all the customer information that you have in your database. If you do not have any customers in your database, test this with another controller.

**At least one person needs to approve pull request before merged to master**
